cmd/create/execution.go

```diff
@@ -0,0 +1,200 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+
+	"github.com/google/uuid"
+)
+
+const (
+	executionShort = "Create execution resources"
+	executionLong  = `
+Create the executions for given workflow/task in a project and domain.
+
+There are three steps in generating an execution.
+
+- Generate the execution spec file.
+- Update the inputs for the execution if needed.
+- Run the execution by passing in the generated yaml file.
+
+The spec file should be generated first and then run the execution using the spec file.
+This would use the latest version of the task core.advanced.run_merge_sort.merge if version flag is not passed.
+Notice the source and target domain/projects can be different.
+
+::
+
+ bin/flytectl create execution  --sourceDomain development  --sourceProject flytectldemo --targetDomain development --targetProject flytesnacks --task core.advanced.run_merge_sort.merge   --genExecSpecFile
+
+The generated file would look similar to this
+
+.. code-block:: yaml
+
+	 genExecSpecFile: true
+	 iamRoleURN: 'example: arn:aws:iam::12345678:role/defaultrole'
+	 inputs:
+	   sorted_list1:
+	   - 0
+	   sorted_list2:
+	   - 0
+	 kubeServiceAcct: ""
+	 sourceDomain: development
+	 sourceProject: flytectldemo
+	 targetDomain: development
+	 targetProject: flytectldemo
+	 task: core.advanced.run_merge_sort.merge
+	 version: ""
+
+
+The generated file can be modified to change the input values.
+
+.. code-block:: yaml
+
+	 genExecSpecFile: true
+	 iamRoleURN: 'example: arn:aws:iam::12345678:role/defaultrole'
+	 inputs:
+	   sorted_list1:
+	   - 2
+	   - 4
+	   - 6
+	   sorted_list2:
+	   - 1
+	   - 3
+	   - 5
+	 kubeServiceAcct: ""
+	 sourceDomain: development
+	 sourceProject: flytectldemo
+	 targetDomain: development
+	 targetProject: flytectldemo
+	 task: core.advanced.run_merge_sort.merge
+	 version: ""
+
+And then can be passed through the command line.
+
+::
+
+ bin/flytectl create execution --file core.advanced.run_merge_sort.merge.execution_spec.yaml
+
+Usage
+`
+)
+
+//go:generate pflags ExecutionConfig --default-var executionConfig
+
+// ProjectConfig Config hold configuration for project create flags.
+type ExecutionConfig struct {
+	File            string                 `json:"file,omitempty" pflag:",file for the execution params.If not specified defaults to <<workflow/task>_name>.inputs.yaml"`
+	GenExecSpecFile bool                   `json:"genExecSpecFile,omitempty" pflag:",bool flag to indicate the generation of execution spec file."`
```

Comment:
just move to `flytectl get task` and `flytectl get launchplan`

Reply:
Done.